app.js

```diff
@@ -410,24 +442,42 @@ const tslint = {
     printDone('Installing HammerJS...');
   }
 
+  if (answers.git) {
+    printMsg('Linkining to git...');
+    try {
```

Comment:
I like the idea here.
But that could be in the "when" property in the inquirer.prompt so the questionwont even be asked if Git is not installed